### DISCUSSION TITLE
关于 x264 的 aq-mode 4

### DISCUSSION THREAD

**Discussion by op200** [vcb-s/guides#18] [Q&A]
https://github.com/vcb-s/guides/blob/af7cb2423d49c37c16da6d831afaf1aeab414867/Basics/%5B07%5D%20%E8%A7%86%E9%A2%91%E7%BC%96%E7%A0%81%E5%99%A8%E5%9F%BA%E7%A1%80/README.md?plain=1#L877

教程中提到现在 x264 中也有 ` aq-mode 4` ，但官方版的 x264 最高只有 `aq-mode 3`，教程中说的是魔改版的吗

**Comment by dtlnor** [comment]
教程中使用的工具以 https://github.com/vcb-s/guides/blob/master/Basics/%5B03%5D%20%E7%A5%9E%E4%B8%80%E6%A0%B7%E7%9A%84%E5%B7%A5%E5%85%B7%E4%BB%AC/README.md 這裡提及的為準

**Reply by op200** [reply]
了解了，感谢解答
